charts/comet-site-v1/templates/main-ingress.yaml

```diff
@@ -13,11 +13,24 @@ metadata:
 spec:
   ingressClassName: nginx
   rules:
-{{- if .Values.cdn.enabled }}
+  {{- if .Values.domains }}
```

Comment:
Do we really need support for domains & domain?

Reply:
Yes. This is neccessary to support Comet < 7 and Comet 7 in the same chart.
Domain will be removed in the next major version, but we wanted to support all current comet versions with the first GitHub version